main/ejml-core/src/org/ejml/data/DMatrixSparseCSC.java

```diff
@@ -212,16 +212,20 @@ public double unsafe_get( int row, int col, double fallBackValue ) {
      * @param col column coordinate
      * @return nz_row index or -1 if the element does not exist
      */
-    public int nz_index( int row, int col ) {
```

Comment:
For better or worse the project code style is  have spaces at the beginning and end of arguments. At some point I'm going to add a CI check. If you use IntelliJ it should be properly with the correct formatting if you tell it to clean up the code.

Reply:
Attempting to enforce a constant style is very recent. I've attempted to configure it to only be mildly opinionated. This file I recently enforced the formatting.